### DISCUSSION TITLE
problem with create_make_pool_snapshot_data_function(networked_components);

### DISCUSSION THREAD

**Discussion by RallyTronics** [xissburg/edyn#114] [Q&A] [answered]
I am running a debug build in VS2022 of my networked Edyn application.  After updating to the latest I get an exception here:

```
std::unique_ptr<pool_snapshot_data>(*g_make_pool_snapshot_data)(component_index_type) =
    create_make_pool_snapshot_data_function(networked_components);

}

```
It is trying to allocate 1437153025663 bytes.    It does this on both my client and server applications.  Any thoughts?

**Comment by xissburg** [comment]
That looks like some deep UB. Are you registering networked components? Have you tried rebuilding everything?

**Comment by RallyTronics** [comment, marked as answer]
I think I figured it out.  The lib was built for double precision, but the application was built for single precision.
